diff --git a/user+group+s3bucket b/user+group+s3bucket
new file mode 100644
--- /dev/null
+++ b/user+group+s3bucket
@@ -0,0 +1,15 @@
+provider "aws" {
+               region = "us-east-2"
+}
+resource "aws_iam_group" "group" {
+    name = "group1"
+  
+} 
+resource "aws_iam_user" "user" {
+    name = "idli"
+  }
+  resource "aws_s3_bucket" "newbucket"{
+    name = "idli_ki_bucket1999"
+    
+    
+  }
\ No newline at end of file